website/docs/r/route_table_association.html.markdown

```diff
@@ -32,3 +32,17 @@ In addition to all arguments above, the following attributes are exported:
 
 * `id` - The ID of the association
 
+## Import
+
+~> **NOTE:** Attempting to associate a route table with a subnet, where either
```

Comment:
We should be covering this in the future with https://github.com/terraform-providers/terraform-provider-aws/issues/9223 😄

Reply:
You want the note removed?